toshiba/swanky/default.nix

```diff
@@ -20,7 +20,7 @@ in
   system.replaceRuntimeDependencies = [{
     original = pkgs.alsa-lib;
 
-    replacement = pkgs.alsa-lib.overrideAttrs (super: {
+    replacement = pkgs.alsa-lib.overrideAttrs (_super: {
```

Comment:
This also looks like a mistake

Reply:
Not a mistake. deadnix renames variables if they are unused.

Reply:
Thats new? I thought it just replaced them with _